<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>800</width>
    <height>600</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <layout class="QGridLayout" name="gridLayout">
    <item row="0" column="0">
     <widget class="QTableView" name="tableView">
      <property name="alternatingRowColors">
       <bool>true</bool>
      </property>
      <property name="selectionMode">
       <enum>QAbstractItemView::SingleSelection</enum>
      </property>
      <property name="selectionBehavior">
       <enum>QAbstractItemView::SelectRows</enum>
      </property>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>800</width>
     <height>21</height>
    </rect>
   </property>
   <widget class="QMenu" name="menu">
    <property name="title">
     <string>Файл</string>
    </property>
    <addaction name="addStudent"/>
    <addaction name="removeStudent"/>
    <addaction name="saveXML"/>
    <addaction name="readXML"/>
   </widget>
   <addaction name="menu"/>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
  <action name="addStudent">
   <property name="text">
    <string>Добавить студента</string>
   </property>
  </action>
  <action name="removeStudent">
   <property name="text">
    <string>Удалить студента</string>
   </property>
  </action>
  <action name="saveXML">
   <property name="text">
    <string>Сохранить в XML</string>
   </property>
  </action>
  <action name="readXML">
   <property name="text">
    <string>Открыть XML</string>
   </property>
  </action>
 </widget>
 <resources/>
 <connections/>
</ui>
